src/lib/cache.ts

```diff
@@ -1,28 +1,24 @@
 import NodeCache from 'node-cache';
 
-// Note, this is heavyweight implementation as it constructs a new cache for
-// each invocation. The assumption is that we'll only ever cache a few values;
-// if we end up wanting to cache lots of things, we should refactor to *share*
-// caches.
+const cache = new NodeCache();
```

Comment:
the cache is now shared application-wide, and a key must be supplied on each call